### DISCUSSION TITLE
Entries made in the web gui doesn't show up in the app

### DISCUSSION THREAD

**Discussion by BaldFabi** [dani-garcia/vaultwarden#5250] [Q&A]
I just freshly deployed vaultwarden in my homelab using the official image. Additionally I downloaded the current version of the Bitwarden app on my iPhone.
After logging in and confirming that everything works regarding my account I tried to create some test entries.
When I create an entry within the web gui the entry will be saved and shown in gui but not in the app.
But if I create an entry within the app the entry shows up in the web gui and everything looks like it should.
Which confuses me even more is if I modify the entry in the web gui (which was created via app) I don't see any changes in the app, only in the web gui.

In short:
* Creating or modifying entries in the web gui don't show up in the app
* All changes and states are correct in the web gui if the entries were created or modified in the app

The logs doesn't show anything except the typical access log entries.
But I wonder why not even a manual refresh in the app triggers an access log entry on the server.

I've searched for similar problems or issues but couldn't find anything.
I'm sure I'm missing something if I'm the only one who's having that problem 😄

**Comment by BaldFabi** [comment]
I just found the reason for this behavior. You need to change some settings in the app and thats the reason why I didn't see any requests in the log of the server.
After a week I just opened the app and was surprised that the entries I was missing are now displayed in the app and thought the app must now has somehow synced and that was the case.
After a some time I found in the settings of the app under "other" the option "Allow sync on refresh" which was not activated... for whatever reason...
